projects/libsodium/secret_key_auth_fuzzer.cc

```diff
@@ -0,0 +1,22 @@
+#include <string>
+extern "C" {
+  #include <sodium.h>
+}
+
+using std::string;
+
+extern "C" int LLVMFuzzerTestOneInput(const unsigned char *data, size_t size) {
+  if (sodium_init() == -1) {
+    return 0;
+  }
+
+  unsigned char key[crypto_auth_KEYBYTES];
+  unsigned char mac[crypto_auth_BYTES];
+
+  crypto_auth_keygen(key);
```

Comment:
Don't generate random things in fuzzers, it should either come from the input, or be a constant.